resource_pack/beet.yaml

```diff
@@ -0,0 +1,31 @@
+id: gm4_resource_pack
+name: Resource Pack
+version: 1.1.X
+
+resource_pack:
+  load:
+    pack.png: pack.png
+    # other files are inherited by the full-project build
+
+pipeline:
+  - dev_warning
```

Comment:
Should this not be `resource_pack.dev_warning`, like other module plugins?

Reply:
Within the scope of this subproject (the `resource_pack` folder), those two are identical. Beet uses python imports for this so it searches the local scope before escalating upward

Reply:
Is this just for preventing people running `beet build` in the `resource_pack` folder? We don't have that for any other module and I don't know of anyone that has gotten confused by it. It should just be clearly explained in the docs how to build the resource pack.

Reply:
This is to prevent people from `beet dev resource_pack` which is otherwise valid and produces an empty pack. It's structured like a module and it's reasonable to think it would work like one.

Reply:
Oh interesting, I did indeed expect that to work. How do you currently build the resource pack? Is `beet-release` the only way?

Reply:
Yes, `beet-release` is the only way to get a full build of the RP. This is because it needs to go through the normal beet build process for each pack to load and process their assets. The RP is then assembled by the resource pack resulting from beet merging every module's individual resource pack.

In normal development, devs should only need to interact with the assets associated with their current module(s). `beet dev` is setup to output and link a smaller resource pack produced by just the modules listed. For example, `beet dev apple_trees zauber_cauldrons` will produce a single `gm4_resource_pack` in the out directory that contains both apple trees and zauber resource pack assets.